plugins/importexport/datacite/scheduledTasks.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  * plugins/importexport/datacite/scheduledTasks.xml
+  *
+  * Copyright (c) 2013-2015 Simon Fraser University Library
+  * Copyright (c) 2003-2015 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * DataCite Plugin Automatic Deposit
+  *
+  * This file describes the schedule task that will send deposits to DataCite in the background.
+  *
+  -->
+
+<!DOCTYPE scheduled_tasks SYSTEM "../../../lib/pkp/dtd/scheduledTasks.dtd">
+
+<scheduled_tasks>
+	<task class="plugins.importexport.datacite.DataciteInfoSender">
+		<descr>Send DOI deposit to DataCite server.</descr>
+		<frequency minute="0"/>
```

Comment:
Ok, I checked the AcronPlugin code and it doesn't allow 0 values. It uses
24 hours instead. So you can leave it that way. :)

2015-09-09 11:02 GMT-03:00 bozana <notifications@github.com>:

> In plugins/importexport/datacite/scheduledTasks.xml
> <https://github.com/pkp/ojs/pull/594#discussion_r39045453>:
>
> > +  * Copyright (c) 2013-2015 Simon Fraser University Library
> > +  * Copyright (c) 2003-2015 John Willinsky
> > +  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
> > +  *
> > +  * DataCite Plugin Automatic Deposit
> > +  *
> > +  * This file describes the schedule task that will send deposits to DataCite in the background.
> > +  *
> > +  -->
> > +
> > +<!DOCTYPE scheduled_tasks SYSTEM "../../../lib/pkp/dtd/scheduledTasks.dtd">
> > +
> > +<scheduled_tasks>
> > +	<task class="plugins.importexport.datacite.DataciteInfoSender">
> > +		<descr>Send DOI deposit to DataCite server.</descr>
> > +		<frequency minute="0"/>
>
> OK :-) Should this be then also changed for the CrossRef plugin?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pkp/ojs/pull/594/files#r39045453>.
>